server.md

```diff
@@ -199,7 +203,7 @@ Example response:
   - `description` is a string that is hardcoded in the `state.ini` file and can be edited there.
   - `creation_time` is the creation time in seconds since 1970-01-01, hardcoded in the `state.ini` file, where it can be edited.
  - `state_path` is the state file path of the server (the value of the environment variable TABPY_STATE_PATH at the time the server was started).
-  - `server_version` is a hardcoded string provided by the server (defined in `server/common/config.py`). Clients can use this information for compatibility checks.
+  - `server_version` is a string provided by the server and is defined by the variable TABPY_SERVER_VERSION in either `server/common/config.py` or as an environment variable. Clients can use this information for compatibility checks.
```

Comment:
This should now read "...and is defined by the variable TABPY_SERVER_VERSION in either `server/common/config.py.template`   ...'

after merging with #92